### DISCUSSION TITLE
How to separate config from downloads

### DISCUSSION THREAD

**Discussion by dundunn** [haugene/docker-transmission-openvpn#2198] [Q&A] [answered]
For whatever reason the folders aren't separated by itself. I just want to have my config file on my SSD and my downloads on my HDD. 

I tried the following:
```
-v /mnt/storage:/storage \
-v /home/dun/.transmission:/data \
```

But this only leads to Transmission resetting the settings.json no matter what's in there.

**Comment by pkishino** [comment, marked as answer]
are you using latest or dev branch? dev branch has updated default config to /config but you can do the same on latest branch too.
for better clarity you can use the following:
```
-v /mnt/storage:/data \
-v /home/dun/.transmission:/config \
```

then additionally you need to specify
```
-e TRANSMISSION_HOME=/config/transmission-home
-e TRANSMISSION_DOWNLOAD_DIR=/data/completed
```
please see the documentation for further descriptions
